specs/binding-mqtt-kafka.spec/src/main/resources/META-INF/zilla/mqtt_kafka.idl

```diff
@@ -30,4 +30,22 @@ scope mqtt_kafka
          int8 length;
          int16[length] packetIds = null;
     }
+
+    scope event
+    {
+        enum MqttKafkaEventType (uint8)
+        {
+            RESET_MQTT_CONNECTION (1)
```

Comment:
In general, you should think about these events as a message to devops so they can correct the problem.

For example, if the sessions topic is not configured correctly, i.e. cleanup policy is not `compact`, then that should be expressed here, rather than saying that the mqtt connection was reset, as the devops does not know how to read this event log and correct the problem in Kafka.